tests/questionpy_common/test_manifest.py

```diff
@@ -129,7 +129,7 @@ def test_valid_api_version(version: str) -> None:
         "1",
         ".1",
         "1.",
-        "01.0" "0.01",
+        "01.00.01",
```

Comment:
Hier und weiter unten haben anscheinend die Kommata gefehlt. :D Das Resultat ist zwar auch nicht valid, aber das sollte ursprünglich nicht getestet werden.
```suggestion
        "01.0",
        "0.01",
```

Reply:
0b31383

Reply:
[Das](https://github.com/questionpy-org/questionpy-server/pull/152/files#diff-03dcaa3466be74adecf625950f90054885bc45b2dcceb88f021d4c8c291c9a99L141-R142) ist auch noch betroffen.